app/controllers/sessions_controller.rb

```diff
@@ -17,6 +17,10 @@ def index
       return redirect_to original_fullpath if original_fullpath.present?
 
       redirect_to summaries_url
+    else
+      # if the symphony client returns no user we might want to redirect to a page that
+      # says that they don't have a record?
+      redirect_to '/500'
```

Comment:
we can redirect to a custom error page for this case as you mentioned @whereismyjetpack, we can add that in a later PR 👍 I think it is good for now, we all would love to see this PR deployed.